nflow-tests/src/test/java/io/nflow/tests/SkipAutoStartTest.java

```diff
@@ -0,0 +1,26 @@
+package io.nflow.tests;
+
+import static org.junit.Assert.assertNotNull;
+
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import io.nflow.tests.runner.NflowServerRule;
+
+public class SkipAutoStartTest extends AbstractNflowTest {
+
+  @ClassRule
+  public static NflowServerRule server = new NflowServerRule.Builder()
```

Comment:
Would be better to have some comment/explicit code that configures non working database connection.